main/ZgatewayBT.ino

```diff
@@ -998,22 +998,40 @@ void launchBTDiscovery(bool overrideDiscovery) {
         std::string brand = decoder.getTheengAttribute(p->sensorModel_id, "brand");
         std::string model = decoder.getTheengAttribute(p->sensorModel_id, "model");
         std::string model_id = decoder.getTheengAttribute(p->sensorModel_id, "model_id");
+        String discovery_topic = String(subjectBTtoMQTT) + "/" + macWOdots;
+        if (p->sensorModel_id == TheengsDecoder::BLE_ID_NUM::NUT ||
+            p->sensorModel_id == TheengsDecoder::BLE_ID_NUM::MIBAND ||
+            p->sensorModel_id == TheengsDecoder::BLE_ID_NUM::TAGIT ||
+            p->sensorModel_id == TheengsDecoder::BLE_ID_NUM::TILE ||
+            p->sensorModel_id == TheengsDecoder::BLE_ID_NUM::TILEN ||
+            p->sensorModel_id == TheengsDecoder::BLE_ID_NUM::ITAG ||
+            p->sensorModel_id == TheengsDecoder::BLE_ID_NUM::RUUVITAG_RAWV1 ||
+            p->sensorModel_id == TheengsDecoder::BLE_ID_NUM::RUUVITAG_RAWV2) {
+          String tracker_name = String(model_id.c_str()) + "-tracker";
+          String tracker_id = macWOdots + "-tracker";
+          createDiscovery("device_tracker",
+                          discovery_topic.c_str(), tracker_name.c_str(), tracker_id.c_str(),
+                          will_Topic, "occupancy", "{% if value_json.rssi -%}home{%- else -%}not_home{%- endif %}",
```

Comment:
```suggestion
                          will_Topic, "occupancy", "{% if value_json.get('rssi') -%}home{%- else -%}not_home{%- endif %}",
```

Using json_value.rssi creates a warning in HA when rssi field is absent. IMHO it is better to use json_value.get('rssi') instead.

> Template variable warning: 'dict object' has no attribute 'rssi' when rendering '{% if value_json.rssi -%}home{%- else -%}not_home{%- endif %}'

Reply:
Thanks